IoT_master/tools/camera_initialize.py

```diff
@@ -0,0 +1,37 @@
+import cv2
+
+class Camera:
```

Comment:
Can you please add docstring on what this class does?
It should be formatted:
```
“””Short description on what this class does
and what is it’s purpose.
“””
```

Reply:
Done.